expression/builtin_string.go

```diff
@@ -1548,7 +1548,65 @@ type builtinInsertFuncSig struct {
 
 // See https://dev.mysql.com/doc/refman/5.6/en/string-functions.html#function_insert
 func (b *builtinInsertFuncSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("insert")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return types.Datum{}, errors.Trace(err)
+	}
+
+	// INSERT(str,pos,len,newstr)
```

Comment:
Remove line1556 ~ line 1561, no need to check args here
They will be checked in baseFunctionClass.verifyArgs().